ui/components/app/assets/nfts/nfts-items/nfts-items.js

```diff
@@ -243,15 +262,12 @@ export default function NftsItems({
             {nfts.map((nft, i) => {
               const { image, address, tokenId, name, imageOriginal, tokenURI } =
                 nft;
-              const nftImage = getAssetImageURL(
-                imageOriginal ?? image,
-                ipfsGateway,
-              );
               const nftImageAlt = getNftImageAlt(nft);
-              const isImageHosted = image?.startsWith('https:');
-              const nftImageURL = imageOriginal?.startsWith('ipfs')
-                ? nftImage
-                : image;
+              const isImageHosted =
+                image?.startsWith('https:') || image?.startsWith('http:');
```

Comment:
Same same, feel free to just resolve all of these if you opt to not make this change